### ISSUE TITLE
does not log in to the server

### ISSUE THREAD

**Issue by vaflya11** [AllTheMods/ATM-8#1100]
![H7XzXbbQDic](https://user-images.githubusercontent.com/128414642/226429935-3e54c2d6-b590-41bf-9ab9-12156a8e8a5a.jpg)
please help solve the problem
the versions of the game and forge are the same

**Comment by Mitchell5200** [comment]
client and server latest.logs via https://mclo.gs/ please

**Comment by vaflya11** [comment]
> client and server latest.logs via https://mclo.gs/ please

https://mclo.gs/4LJhs and https://mclo.gs/hxcAJDe

thank you in advance

**Comment by Mitchell5200** [comment]
Okay, Looking at the server log, it seems mostly fine, apart from it running in Offline mode,
Looking at the client, you are missing several mods that are normally in the client install, and you are missing many additional files that are needed both serverside and client side.
This combination leads me to the conclusion that whatever cracked launcher you're using only ripped the mods folder from the server pack, or you only imported the mods from the server pack to the client. 

TLDR, your main issue is a improper client installation issue, your second issue is using a cracked launcher, which we do not support
